rpc/block_test.go

```diff
@@ -677,130 +923,31 @@ func TestCaptureUnsupportedBlockTxn(t *testing.T) {
-				InvokeTxnV1: InvokeTxnV1{
-					Type:          "INVOKE",
-					Version:       TransactionV1,
-					Nonce:         utils.TestHexToFelt(t, "0x33"),
-					MaxFee:        utils.TestHexToFelt(t, "0x1bad55a98e1c1"),
-					SenderAddress: utils.TestHexToFelt(t, "0x3543d2f0290e39a08cfdf2245f14aec7dca60672b7c7458375f3cb3834e1067"),
-					Signature: []*felt.Felt{
-						utils.TestHexToFelt(t, "0x1"),
-						utils.TestHexToFelt(t, "0x7bc0a22005a54ec6a005c1e89ab0201cbd0819621edd9fe4d5ef177a4ff33dd"),
-						utils.TestHexToFelt(t, "0x13089e5f38de4ea98e9275be7fadc915946be15c14a8fed7c55202818527bea"),
-					},
-					Calldata: []*felt.Felt{
-						utils.TestHexToFelt(t, "0x1"),
-						utils.TestHexToFelt(t, "0x517567ac7026ce129c950e6e113e437aa3c83716cd61481c6bb8c5057e6923e"),
-						utils.TestHexToFelt(t, "0xcaffbd1bd76bd7f24a3fa1d69d1b2588a86d1f9d2359b13f6a84b7e1cbd126"),
-						utils.TestHexToFelt(t, "0x6"),
-						utils.TestHexToFelt(t, "0x5265706f73736573734275696c64696e67"),
-						utils.TestHexToFelt(t, "0x4"),
-						utils.TestHexToFelt(t, "0x5"),
-						utils.TestHexToFelt(t, "0x1b48"),
-						utils.TestHexToFelt(t, "0x1"),
-						utils.TestHexToFelt(t, "0xe52"),
-					},
-				},
-			},
-		},
-	}
-
-	testSet := map[string][]testSetType{
-		"mock": {},
-		"testnet": {
-			{
-				BlockID:                     WithBlockNumber(64159),
-				ExpectedError:               nil,
-				want:                        &fullBlockSepolia64159,
-				LookupTxnPositionInExpected: 0,
-				LookupTxnPositionInOriginal: 4,
-			},
-		},
-		"mainnet": {},
-	}[testEnv]
-
-	require := require.New(t)
-	for _, test := range testSet {
-		spy := NewSpy(testConfig.provider.c)
-		testConfig.provider.c = spy
-		blockWithTxsInterface, err := testConfig.provider.BlockWithTxs(context.Background(), test.BlockID)
-		require.NoError(err, "Unable to fetch the given block.")
-
-		blockWithTxs, ok := blockWithTxsInterface.(*Block)
-		require.True(ok, "Failed to assert the Interface as *Block.")
-		require.Equal(blockWithTxs.BlockHash.String()[:2], "0x", "Block Hash should start with \"0x\".")
-		require.NotEqual(len(blockWithTxs.Transactions), 0, "The number of transaction should not be 0.")
-
-		invokeV1Want, ok := (*test.want).Transactions[test.LookupTxnPositionInExpected].(BlockInvokeTxnV1)
-		require.True(ok, "Expected invoke v1 transaction.")
-
-		invokeV1Block, ok := blockWithTxs.Transactions[test.LookupTxnPositionInOriginal].(BlockInvokeTxnV1)
-		require.True(ok, "Expected invoke v1 transaction.")
-
-		require.Equal(invokeV1Want.TransactionHash.String(), invokeV1Block.TransactionHash.String(), "Expected equal TransactionHash.")
-		require.Equal(invokeV1Want.InvokeTxnV1.MaxFee.String(), invokeV1Block.InvokeTxnV1.MaxFee.String(), "Expected equal maxfee.")
-		require.Equal(invokeV1Want.InvokeTxnV1.Calldata[1].String(), invokeV1Block.InvokeTxnV1.Calldata[1].String(), "Expected equal calldatas.")
-	}
-}
-
 // TestStateUpdate is a test function for the StateUpdate method.
```

Comment:
The mock test data here is from the mock network. Can we update it to use sepolia data please?